frame/staking/src/lib.rs

```diff
@@ -782,6 +782,36 @@ impl<T: Config> SessionInterface<<T as frame_system::Config>::AccountId> for T w
 	}
 }
 
+pub trait EraPayout<Balance> {
```

Comment:
Also, would be much appreciated if we provide a dummy `impl EraPayout for ()` for testing and get rid of all the verbose `pallet_staking_reward_curve::build!` in testing of pallets that won't care about this, such as babe and session. 

(Something that I've also requested previously from @thiolliere)